src/form/Input.js

```diff
@@ -84,8 +84,7 @@
             w.blur(w);
         });
         var context = this;
-        this._inputElement.on("change", function (w) {
-
+        this._inputElement.on("keyup", function (w) {
```

Comment:
I am nervous that this won't work for some of the esoteric inputs (like html-color)